src/Pororoca.Desktop/Localization/i18n_keys.json

```diff
@@ -1,291 +1,292 @@
+    "WebSocketClientMessage/ContentFileSearchFile",
+    "WebSocketClientMessage/ContentFileSrcPathWatermark",
+    "WebSocketClientMessage/ContentModeFile",
+    "WebSocketClientMessage/ContentModeRaw",
+    "WebSocketClientMessage/ContentRawWatermark",
+    "WebSocketClientMessage/DisableCompressionExplanation",
+    "WebSocketClientMessage/DisableCompressionForThisMessage",
+    "WebSocketClientMessage/DisableCompressionForThisMessageTip",
+    "WebSocketClientMessage/MessageTypeBinary",
+    "WebSocketClientMessage/MessageTypeClose",
+    "WebSocketClientMessage/MessageTypeText",
+    "WebSocketClientMessage/NewMessage",
+    "WebSocketClientMessage/RawContentJsonSyntax",
+    "WebSocketClientMessage/RawContentOtherSyntax",
+    "WebSocketConnection/CancelConnect",
+    "WebSocketConnection/CancelConnectToolTip",
+    "WebSocketConnection/CancelDisconnect",
+    "WebSocketConnection/CompressionClientContextTakeover",
+    "WebSocketConnection/CompressionClientMaxWindowBits",
+    "WebSocketConnection/CompressionServerContextTakeover",
+    "WebSocketConnection/CompressionServerMaxWindowBits",
+    "WebSocketConnection/Connect",
+    "WebSocketConnection/ConnectToolTip",
+    "WebSocketConnection/Connected",
+    "WebSocketConnection/ConnectionException",
+    "WebSocketConnection/ConnectionExceptionWatermark",
+    "WebSocketConnection/ConnectionRequestHeaderAdd",
+    "WebSocketConnection/ConnectionRequestHeaderEnabled",
+    "WebSocketConnection/ConnectionRequestHeaderName",
+    "WebSocketConnection/ConnectionRequestHeaderRemove",
+    "WebSocketConnection/ConnectionRequestHeaderValue",
+    "WebSocketConnection/ConnectionRequestOptions",
+    "WebSocketConnection/ConnectionRequestSubprotocolAdd",
+    "WebSocketConnection/ConnectionRequestSubprotocolEnabled",
+    "WebSocketConnection/ConnectionRequestSubprotocolName",
+    "WebSocketConnection/ConnectionRequestSubprotocolRemove",
+    "WebSocketConnection/CustomAuth",
+    "WebSocketConnection/Disconnect",
+    "WebSocketConnection/DisconnectToolTip",
+    "WebSocketConnection/EnableCompression",
+    "WebSocketConnection/NewConnection",
+    "WebSocketConnection/OptionCompression",
+    "WebSocketConnection/OptionHeaders",
+    "WebSocketConnection/OptionSubprotocols",
+    "WebSocketConnection/SectionTitle",
+    "WebSocketConnection/UrlWatermark",
+    "WebSocketExchangedMessageDetail/ContentWatermark",
+    "WebSocketExchangedMessageDetail/MessageTypeWatermark",
+    "WebSocketExchangedMessageDetail/SaveToFile",
+    "WebSocketExchangedMessageDetail/SaveToFileToolTip",
+    "WebSocketExchangedMessageDetail/SectionTitle",
+    "WebSocketExchangedMessages/BinaryContentDescriptionFormat",
+    "WebSocketExchangedMessages/ClosingMessageContentDescriptionFormat",
+    "WebSocketExchangedMessages/FromClientToServer",
+    "WebSocketExchangedMessages/FromServerToClient",
+    "WebSocketExchangedMessages/InstantDescriptionFormat",
+    "WebSocketExchangedMessages/MessagesExchangedTab",
```

Comment:
Let's keep the `"WebSocketExchangedMessages/SectionTitle"` key here instead of reusing `"HttpResponse/SectionTitle"`, even if they will have the same values in the language files, this is for better separation in case we want to change them